### PR TITLE
Adds "Box, Box!"

### PR DESCRIPTION
**Pull request by Rafee-M**
[Box, Box!](https://github.com/BrightDV/BoxBox)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -684,6 +684,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 
 ### • Sports & Health
 
+* [**Box, Box!**](https://github.com/BrightDV/BoxBox) <sup>**[[F-Droid](https://f-droid.org/packages/org.brightdv.boxbox/)]**</sup>
```
Comment (offa) [inline README.md:687]: Please use the [`…/app/…` link format](https://github.com/offa/android-foss/blob/master/CONTRIBUTING.md#f-droid-links).

**Review by offa — CHANGES_REQUESTED** [review]

**Comment by offa** [comment]
Thanks for your PR. *Racing* category fits better here 🏎️.

File: README.md (hunk 1 above)
Comment (Rafee-M) [reply README.md]: My bad, didn't notice it

**Comment by Rafee-M** [comment]
> Thanks for your PR. _Racing_ category fits better here 🏎️.

But that's sub category under _Games_ and it's not really a game. More of a racing event tracker. Then again _Sports & Health_ is about fitness and health.

I am unclear here, please advise

**Review by offa — APPROVED** [review]

**Pull request merged**